app/graphql/types/git/commit_type.rb

```diff
@@ -94,9 +94,19 @@
     end
 
     resolve(lambda do |commit, arguments, _context|
-      GitFile.new(commit,
-                  arguments['path'],
-                  load_all_data: arguments['loadAllData'])
+      git_file = GitFile.new(commit,
+                             arguments['path'],
+                             load_all_data: arguments['loadAllData'])
+      git_file if git_file.exist?
+    end)
+  end
+
+  field :lsFiles, !types[!types.String] do
```

Comment:
What do you think about returning a list of files here instead of just a list of paths? Would that be computationally expensive to do?